perf-delta.nix

```diff
@@ -36,7 +36,7 @@ let
         do
           # ignore all errors
           echo -n $file
-          if timeout 10s moc $file -no-check-ir -ref-system-api -o $file.wasm 2>/dev/null
+          if timeout 10s moc $file --force-gc --compacting-gc -no-check-ir -ref-system-api -o $file.wasm 2>/dev/null
```

Comment:
@osa1 should we leave this `--compacting-gc` in? I guess it will be the default, so we should test it with priority...

Reply:
I'll leave this in and revisit it as soon as #2930 gets ripe.